R/dbplyr.R

```diff
@@ -0,0 +1,282 @@
+#' @include DBI.R
+NULL
+
+temporaryErrorMessage <- paste0("You cannot use `temporary = TRUE`",
```

Comment:
I have mixed feelings. They feel inelegant to me, but users tend to really appreciate them.